Add READMES to quark script showcases

zinwang opened the pull request:
# Description

This PR aims to update Quark Script to the repo and complete the README file of each Quark Script showcase.

# File Changes
 + Add CWE-327 Quark Script 
 + Add a README file to each showcase

haeter525 commented on CWE-94/README.md:
```diff
@@ -0,0 +1,66 @@
+
+First, we design a detection rule loadExternalCode.json to spot on behavior uses method createPackageContext. Then, we find the caller method who calls the createPackageContext. Finally, we check if method checkSignatures is called in the caller method for verification.
+
+## Quark Scipt: CWE-94.py
```
Please correct the spelling here.

haeter525 commented on CWE-312/README.md:
```diff
@@ -0,0 +1,102 @@
+
+Let’s use this [APK](https://github.com/oversecured/ovaa) and the above APIs to show how Quark script find this vulnerability.
+
+First, we designed a Frida script agent.js to hook the target method and get the arguments when the target method is called. Then we hook the method putString to catch its arguments. Finally, we use Ciphey to check if the arguments are encrypted.
```
The links to the official websites of Frida and Ciphey is missing. Please correct it.

haeter525 reviewed (changes requested):
Hi @zinwang. Thanks for the patch.
I've left one comment on the changes.

Also, I found some content is formatted differently from the document.
1. the APIs and the file name are not marked with backticks.
2. some words are not in bold.

Please fix them as well. Thanks.

zinwang commented:
@haeter525 
I fixed grammar flaws in CWE-94 and added links to CWE-312.
Also, the formats in all the CWEs are the same as the document.
Please have a look! Thank you!

haeter525 reviewed (approved):
Thank @zinwang. LGTM!
